MV.Wildebeest.Core/source/core/resources/co/mv/wb/impl/resource.xsd

```diff
@@ -112,15 +125,15 @@
         </xs:restriction>
     </xs:simpleType>
 
-    <xs:complexType name="Single">
+    <xs:complexType name="SingleAssertionRef">
         <xs:complexContent>
             <xs:extension base="AssertionRef">
                 <xs:attribute name="ref" type="xs:string" use="required"/>
             </xs:extension>
         </xs:complexContent>
     </xs:complexType>
 
-    <xs:complexType name="Selector">
+    <xs:complexType name="SelectorAssertionRef">
```

Comment:
Could we remove the Selector stuff from the schema until we fully implement that feature - just don't want partial stuff in the code as it may cause confusion.

Perhaps we can keep it in a feature branch for now.

Reply:
Sure, I will remove it.   I added that last week when selector is still on my list of tasks to do.